extensions/github-authentication/src/flows.ts

```diff
@@ -72,6 +72,10 @@ interface IFlowTriggerOptions {
 	 * The specific auth provider to use for the flow.
 	 */
 	signInProvider?: GitHubSocialSignInProvider;
+	/**
+	 * Extra parameter to include in the OAuth flow.
+	 */
+	extraParameter?: string;
```

Comment:
what if this was instead:

```suggestion
	/**
	 * Extra parameters to include in the OAuth flow.
	 */
	extraParameters?: Record<string, string>;
```

Reply:
Yeah absolute, and I had it that way originally. I only then figured out that these options are transferred over to the extension host, so we would need some kind of transformation step to convert this type, but maybe you have an idea.